rclpy/package.xml

```diff
@@ -14,9 +14,13 @@
   <depend>rmw_implementation</depend>
   <depend>rcl</depend>
 
+  <exec_depend>ament_index_python</exec_depend>
+
   <test_depend>ament_cmake_nose</test_depend>
+  <test_depend>ament_index_python</test_depend>
```

Comment:
I opened https://github.com/ament/ament_tools/pull/86 with what I believe to be the appropriate fix.